### PR TITLE
doc: prepare new release 3.7.0

### PR DESCRIPTION
**Pull request by PhilippeAccorsi**
Related to #4282 

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified
- [x] Original authors of the features included in a multi-feature branch (maintenance fixes -> develop, security fixes -> develop, …) should be part of the reviewers, especially if you encountered merge conflicts.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- No manual, quality tests necessary here

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -1,3 +1,27 @@
+# 3.7.0 / 2021-04-02
+
+### New Features
+
+- You can create your own publications to share informations with other member without creating a specific content (like a blog page)
```
Comment (raphj) [inline changelog.md:5]:
```suggestion
- You can create your own publications to share information with other members without creating a specific content (like a blog page)
```

File: changelog.md [hunk 2]
```diff
@@ -1,3 +1,27 @@
...
+- You can add reactions to contents using emojis like usable in social networking platform
```
Comment (gbonaspetti) [inline changelog.md:6]:
```suggestion
- You can add reactions to contents using emojis like in a social networking platform
```

**Review by gbonaspetti — CHANGES_REQUESTED** [review]

File: changelog.md (hunk 2 above)
Comment (grignards) [inline changelog.md:6]:
```suggestion
- You can add reactions to contents using emojis like you can find on social networking platforms
```

Comment (raphj) [inline changelog.md:6]:
```suggestion
- You can add reactions to contents using emojis the same way as in other social networking platforms or instant messaging tools
```

File: changelog.md [hunk 3]
```diff
@@ -1,3 +1,27 @@
...
+- The activity feed of space is now visible in dashboad and replace recent activity view
```
Comment (grignards) [inline changelog.md:7]:
```suggestion
- The activity feed of a space is now visible in dashboad and replace recent activity view
```

File: changelog.md [hunk 4]
```diff
@@ -1,3 +1,27 @@
...
+
+- Frontend: [#4341](https://github.com/tracim/tracim/issues/4341)
+
+### Breaking Changes
+
+- Database: databases that were created using Tracim v1 need to be updated manually, [more information here](https://github.com/tracim/tracim/issues/4282#issuecomment-801944438) (issue [#4266](https://github.com/tracim/tracim/issues/4266))
+
+### Other Changes
+
+- Reaction feature add big change to mysql/mariadb support: database migration needed for mysql/mariadb with [this tracimcli command line](https://github.com/tracim/tracim/blob/2a6aae281046e7a10bf4c5b24dfc60f81214c709/backend/doc/cli.md#migrate-mysqlmariadb-database-to-utf8mb4-added-for-tracim-37) (issue [#4272](https://github.com/tracim/tracim/issues/4272))
```
Comment (raphj) [inline changelog.md:19]:
```suggestion
- The Reaction feature requires a migration when using a MySQL or MariaDB database. This migration can be applied with [this tracimcli command line](https://github.com/tracim/tracim/blob/2a6aae281046e7a10bf4c5b24dfc60f81214c709/backend/doc/cli.md#migrate-mysqlmariadb-database-to-utf8mb4-added-for-tracim-37) (issue [#4272](https://github.com/tracim/tracim/issues/4272))
```

Comment (grignards) [inline changelog.md:19]:
```suggestion
- This new version requires a special migration for mysql/mariadb (due to the reaction feature): [tracimcli command line](https://github.com/tracim/tracim/blob/2a6aae281046e7a10bf4c5b24dfc60f81214c709/backend/doc/cli.md#migrate-mysqlmariadb-database-to-utf8mb4-added-for-tracim-37) (issue [#4272](https://github.com/tracim/tracim/issues/4272)).
```

File: changelog.md [hunk 5]
```diff
@@ -1,3 +1,27 @@
...
+- API: this API`/api/workspaces/<workspace_id>/contents` now returns an object containing the contents in "items" instead of a direct list and also contains pagination information when the count/page_token parameters are used (issue [#4264](https://github.com/tracim/tracim/issues/4264))
+- Modification of default value used in development.ini: default value for `web.notifications.excluded`as changed to a reduced synonym for user and add filtering for new type `reaction` (issue [#4272](https://github.com/tracim/tracim/issues/4272))
```
Comment (raphj) [inline changelog.md:21]:
```suggestion
- Modification of a default value used in development.ini: default value for `web.notifications.excluded`as changed to a reduced synonym for user and add filtering for new type `reaction` (issue [#4272](https://github.com/tracim/tracim/issues/4272))
```

Comment (grignards) [inline changelog.md:21]:
```suggestion
- Modification of a default value used in `development.ini`: default value for `web.notifications.excluded` has changed to a simpler syntax for user and adds filtering of the new type `reaction` (issue [#4272](https://github.com/tracim/tracim/issues/4272))
```

File: changelog.md [hunk 6]
```diff
@@ -1,3 +1,27 @@
...
+  - On existing Tracim installations, also if using the docker image: you can update your development.ini (use this file [development.ini.sample](backend/development.ini.sample) to compare).
```
Comment (raphj) [inline changelog.md:22]:
```suggestion
  - On existing Tracim installations, also if using the docker image: you may want to update your development.ini configuration file (use this file [development.ini.sample](backend/development.ini.sample) to compare).
```

File: changelog.md (hunk 5 above)
Comment (PhilippeAccorsi) [reply changelog.md:21]: done

File: changelog.md (hunk 4 above)
Comment (PhilippeAccorsi) [reply changelog.md]: I used the grignards wording.

File: changelog.md (hunk 2 above)
Comment (PhilippeAccorsi) [reply changelog.md]: thanks

Comment (PhilippeAccorsi) [reply changelog.md]: thanks

Comment (PhilippeAccorsi) [reply changelog.md]: thanks

File: changelog.md [hunk 7]
```diff
@@ -1,3 +1,27 @@
...
+
+### Breaking Changes
+
+- Database: databases that were created using Tracim v1 need to be updated manually, [more information here](https://github.com/tracim/tracim/issues/4282#issuecomment-801944438) (issue [#4266](https://github.com/tracim/tracim/issues/4266)).
+
+### Other Changes
+
+- This new version requires a special migration for mysql/mariadb (due to the reaction feature): [tracimcli command line](https://github.com/tracim/tracim/blob/2a6aae281046e7a10bf4c5b24dfc60f81214c709/backend/doc/cli.md#migrate-mysqlmariadb-database-to-utf8mb4-added-for-tracim-37) (issue [#4272](https://github.com/tracim/tracim/issues/4272)).
+- API: this API`/api/workspaces/<workspace_id>/contents` now returns an object containing the contents in "items" instead of a direct list and also contains pagination information when the count/page_token parameters are used (issue [#4264](https://github.com/tracim/tracim/issues/4264)).
+- Modification of default value used in development.ini: default value for `web.notifications.excluded`as changed to a reduced synonym for user and add filtering for new type `reaction` (issue [#4272](https://github.com/tracim/tracim/issues/4272)).
```
Comment (grignards) [inline changelog.md:21]:
```suggestion
- Modification of a default value in `development.ini`: the default value for `web.notifications.excluded` has changed to a simple syntax for `user` and adds filtering for new the new notification type `reaction` (issue [#4272](https://github.com/tracim/tracim/issues/4272)).
```

File: changelog.md [hunk 8]
```diff
@@ -1,3 +1,27 @@
...
+  - On existing Tracim installations, also if using the docker image: you may want to update your development.ini configuration file (use this file [development.ini.sample](backend/development.ini.sample) to compare).
```
Comment (grignards) [inline changelog.md:22]:
```suggestion
  - On existing Tracim installations, including the official docker image: you may want to update your `development.ini` configuration file (use this file [development.ini.sample](backend/development.ini.sample) to compare).
```

**Review by gbonaspetti — APPROVED** [review]

**Review by inkhey — APPROVED** [review]

**Pull request merged**